kubeflow/asm/Makefile

```diff
@@ -82,25 +78,32 @@ asmcli-validate:
 	--option legacy-default-ingressgateway \
 	--verbose
 
-
-.PHONY: asmcli-install
-asmcli-install: download-asmcli
-# --fleet_id is optional. If not specified, use project_id
-	./asmcli install \
-	--project_id $(PROJECT) \
-	--cluster_name $(NAME) \
-	--cluster_location $(LOCATION) \
-	--output_dir $(UPSTREAM) \
-	--enable_all \
-	--ca mesh_ca \
-	--custom_overlay ./$(UPSTREAM)/asm/istio/options/iap-operator.yaml \
-	--custom_overlay ./options/ingressgateway-iap.yaml \
-	--option legacy-default-ingressgateway \
-	--verbose
-
 .PHONY: asmcli-precheck
 asmcli-precheck:
 	pushd $(UPSTREAM) && ./istioctl experimental precheck && popd
-
 # Note about install gateways
 # https://cloud.google.com/service-mesh/docs/unified-install/install#install_gateways
+
+.PHONY: download-install-asm-script-legacy
+download-install-asm-script-legacy:
+	curl https://storage.googleapis.com/csm-artifacts/asm/$(INSTALL_ASM_SCRIPT_VERSION) > install_asm;
+	chmod +x install_asm
+# Official documentation of downloading install_asm: https://cloud.google.com/service-mesh/docs/scripted-install/asm-onboarding#downloading_the_script
+# It cannot control the patch version so we won't use this approach for now.
+# cd ./asm && { \
+# 	curl https://storage.googleapis.com/csm-artifacts/asm/install_asm_1.9 > install_asm; \
+# 	curl https://storage.googleapis.com/csm-artifacts/asm/install_asm_1.9.sha256 > install_asm.sha256; \
+# 	sha256sum -c --ignore-missing install_asm.sha256; \
+# 	chmod +x install_asm; \
+# 	cd -;}
+
+.PHONY: install-asm-legacy
+install-asm-legacy: download-install-asm-script-legacy download-asm-package
```

Comment:
Do we still need to install install_asm?

Reply:
Removed